src/command_modules/azure-cli-acr/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+2.2.10
++++++
+* 'az acr check-health' now displays a human friendly Helm version.
+* 'az acr check-health' displays an error message if an obsolete Helm version is detected.
```

Comment:
This history goes to the official release of our CLI: https://pypi.org/project/azure-cli-acr/. Maybe you can make summarize these additions in one phrase as they are on the same command